### PR TITLE
v2

### PR DESCRIPTION
**Pull request by Integralist**

### CODE REVIEW

**Comment by harmony7** [comment]
I did a new build against a few changes I made in the generator.
This seems to have removed a few generated files. These files come relate to "entitled products" API.  I'm thinking maybe the committed version wasn't built against the newest version of api-documentation `5ae30e4b1d0b0c6bf2ba175e695464f5d6b9fdfa` (rebuilding the other 5 clients didn't have this problem).

I've committed a new build.

**Comment by Integralist** [comment]
@harmony7 can you re-review this PR please. @karenetheridge had a PR merged recently into our internal api client generator repo and I've regenerated the client code based off that now. I noticed a bunch of additional files now exist.

File: lib/WebService/Fastly.pm [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+=encoding utf-8
+
+=head1 NAME
+
+WebService::Fastly - an interface to most facets of the [Fastly API](https://developer.fastly.com/reference/api).
+
+=head1 SYNOPSIS
+
+  use WebService::Fastly;
```
Comment (simonwistow) [inline lib/WebService/Fastly.pm:18]: I feel like we need a better synopsis than this.

Maybe

1. Login
2. Get a list of services
3. Clone a service
4. Change something (add Header and/or Domain)
5. Deploy

And maybe something like adding something to a Versionless Dictionary to show the different mode of interaction.

**Comment by harmony7** [comment]
@Integralist I've re-reviewed, I'm good with the changes.
And, I've also made a new build based on another build from the internal generator repo.

File: lib/WebService/Fastly.pm (hunk 1 above)
Comment (harmony7) [reply lib/WebService/Fastly.pm:18]: @simonwistow thanks for the comments.  I'll make a new ticket for this and get these added in the next (2.03) release.

**Comment by harmony7** [comment]
Releasing to CPAN as 2.01.  The release steps will be included in the CI in upcoming PR

**Pull request merged**